src/racf2john.c

```diff
@@ -14,7 +14,19 @@
  * vectors, algorithm details, RACF sample database  and requesting the
  * RACF cracker in the first place.
  *
- * racfdump format => userid:$racf$*userid*deshash  */
+ * racfdump format => userid:$racf$*userid*deshash
+ *
+ ***********************************************************************************
+ * update   4/10/16  BeS - added support to dump racf-kdfaes style password10      *
+ * hashes, in addition to fixing the extraction of the old algorithm so that       *
+ * only active profiles are dumped.                                                *
+ *                                                                                 *
+ *  TODO - add option to dump password history (could be useful) hashes as well    *
```

Comment:
Yep - that's on the way - little more complicated, so I wanted to split up the PRs, but I have it in beta.